dspace-oai/pom.xml

```diff
@@ -16,7 +16,7 @@
         <!-- This is the path to the root [dspace-src] directory. -->
         <root.basedir>${basedir}/..</root.basedir>
         <spring.version>3.2.5.RELEASE</spring.version>
-        <xoai.version>3.2.10</xoai.version>
+        <xoai.version>3.3.0</xoai.version>
```

Comment:
Forgot to add this in my review. Just a note that upgrading XOAI will result in this bug https://jira.lyrasis.org/browse/DS-4404  So, the fix for that bug would also need backporting to 6.x, if the XOAI upgrade is required.

Reply:
Ok, but the DS-4404 have to fixed with patch at https://jira.lyrasis.org/browse/DS-4462

Reply:
This looks like it's been resolved as the bug fixes from DS-4404 and DS-4462 look like they are now part of this PR.